src/Storages/MergeTree/IMergeTreeDataPartWriter.cpp

```diff
@@ -1,9 +1,10 @@
-#include <Storages/MergeTree/IMergeTreeDataPartWriter.h>
-
 #include <Columns/ColumnSparse.h>
-#include <Common/MemoryTrackerBlockerInThread.h>
+#include <Compression/CompressionFactory.h>
+#include <Storages/MergeTree/IMergeTreeDataPartWriter.h>
 #include <Storages/MergeTree/MergeTreeIndexGranularity.h>
+#include <Storages/MergeTree/MergeTreeSettings.h>
 #include <Storages/StorageInMemoryMetadata.h>
+#include <Common/MemoryTrackerBlockerInThread.h>
```

Comment:
I think the order should be main header first, and then the rest of the header.
```suggestion
#include <Storages/MergeTree/IMergeTreeDataPartWriter.h>

#include <Columns/ColumnSparse.h>
#include <Compression/CompressionFactory.h>
#include <Storages/MergeTree/MergeTreeIndexGranularity.h>
#include <Storages/MergeTree/MergeTreeSettings.h>
#include <Storages/StorageInMemoryMetadata.h>
#include <Common/MemoryTrackerBlockerInThread.h>
```

Reply:
Thanks.
I only enabled auto-format for modifications. So it didn't update the include part.

Reply:
It is not that important, I won't hold back the PR because of this.